### PR TITLE
Add columns to the Billing Agreements page

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request adds columns to the Billing Agreements page

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [x] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/sales/billing-agreements.html

### CODE REVIEW

File: src/sales/billing-agreements.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ title: Billing Agreements
 
 The _Billing Agreements_ page lists all PayPal [billing agreements]({% link payment/paypal-billing-agreements.md %}) between your store and its customers. The store administrator can filter the records by the customer or billing agreement information, including billing agreement reference ID, status, and creation date. Each record includes general information about the billing agreement, and all sales orders that have used it as a payment method. A store administrator can view, cancel, or delete customer billing agreements. A canceled billing agreement can be deleted only by the store administrator.
 
+Every time when a customer places an order using a billing agreement, Magento 2 sends the billing agreement reference ID and sales order billing information to PayPal. The relevant information data records in the customer's account.
```
Comment (dobooth) [inline src/sales/billing-agreements.md:7]:
```suggestion
When a customer places an order using a billing agreement, Magento sends the billing agreement reference ID and sales order billing information to PayPal. The relevant information data records in the customer's account.
```
I am not sure what the last sentence here means. Can you rewrite please?

**Review by dobooth — CHANGES_REQUESTED** [review]

**Comment by dobooth** [comment]
@ZhovnirMaria

**Comment by ZhovnirMaria** [comment]
@dobooth Hi, I tried to rephrase the sentence, please let me know if there is any issues😊

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the pull request successfully imported.

**Pull request merged**